staging/src/k8s.io/apiserver/pkg/registry/rest/rest.go

```diff
@@ -332,3 +332,11 @@ type StorageMetadata interface {
 	// it is not nil. Only the type of the return object matters, the value will be ignored.
 	ProducesObject(verb string) interface{}
 }
+
+// StorageVersionProvider is an optional interface that a storage object can
+// impelment if it wishes to disclose its storage version.
+type StorageVersionProvider interface {
+	// StorageVersion returns the storage version, the version to which
+	// objects are converted to before stored to the storage.
```

Comment:
document this can return an empty string (the generic store implementation makes that a possibility)

Reply:
Added.